examples/extensions/uniffi/SerializedObject.toml

```diff
@@ -0,0 +1,16 @@
+# Example of wrapping on the bindings side only
```

Comment:
My vision for this is that we would have a collection of `.toml` extension files, maybe the uniffi repo, maybe a wiki page that linked to gists.  Configuring an extension would mean downloading the  file to their `uniffi` directory and maybe doing some quick configuration like replacing `MyObbject` below.